plugins/resteasy-http/src/main/java/org/radargun/http/service/RESTEasyCacheInfo.java

```diff
@@ -74,11 +77,11 @@ public long getTotalSize() {
                size = Long.parseLong(responseStr);    //if the call was done to nodejs app, then the long size is returned
 
             } catch (NumberFormatException ex) {
-               //If the call was done to Infinispan Server, then the response is the list of all entries
-               //WARNING: As this way the REST API retrieves all keys from the cache and returns them as a response,
-               //this part of block may have an impact on the performance of the stage which rely on this trait.
-               String[] entries = responseStr.split(System.getProperty("line.separator"));
-               size = entries.length;
+               Matcher matcher = Pattern.compile("\"size\":(\\d+)").matcher(responseStr);
```

Comment:
Nitpick: It's more readable and easier to understand if you create a private method with a name which explain what you are doing ou desire here, it's more readable than analyse the pattern, find, parse ...